frameworks/cassandra/universe/config.json

```diff
@@ -55,6 +55,11 @@
           "description": "Enable DC/OS Virtual Networking",
           "type": "boolean",
           "default": false
+        },
+        "log_level": {
```

Comment:
I thought of this and opted not to make a decision by placing no restrictions.  Might be handy to be able do whatever we want in an emergency.  Changing this setting is basically an emergency operation.  So if you want to restrict to an enum I'd go with allowing all options.